test/libsolidity/syntaxTests/userDefinedValueType/in_parenthesis.sol

```diff
@@ -0,0 +1,7 @@
+type MyInt is int;
+function f() pure {
+    (MyInt).wrap;
+    (MyInt).wrap(5);
+    (MyInt).unwrap;
+    (MyInt).unwrap(MyInt.wrap(5));
```

Comment:
Syntactically this is really not interesting - this should be a semantics test!

Reply:
I can move this.

Reply:
Did it here: https://github.com/ethereum/solidity/pull/11914/